src/main/java/nl/tudelft/semgroup4/collision/DefaultCollisionHandler.java

```diff
@@ -230,4 +248,34 @@ public void run() {
         Game.LOGGER.log(LogSeverity.DEBUG, "Collision", "Player picked up a utility");
         util.activate(game.getCurLevel());
     };
+    
+    private final CollisionHandler<HorMovingWall, AbstractWall> horWallWallHandler = 
+            (game, movingWall, abstractWall) -> {
+        int offset = 5;        
+        
+        if (movingWall.getLocX() < abstractWall.getLocX() 
```

Comment:
But the offset has nothing to do with the Wall itself, and everything with collision. So I think it would be odd if you calculate the offset in the Wall object, because it's then responsible for something it doesn't really have anything to do with. 
I think you would want to group your logic concerning something like this.